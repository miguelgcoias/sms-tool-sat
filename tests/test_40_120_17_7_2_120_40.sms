40
108 11 130 6 2 2 1 2 2 2
37 6 43 6 1 1 1 1 1 1
23 17 40 7 4 4 2 1 4 1 1
0 16 16 6 5 2 3 2 3 1
14 17 31 4 4 5 3 5
56 7 63 4 1 2 2 2
58 5 68 1 5
25 6 37 4 1 1 2 2
66 5 71 3 1 2 2
113 15 143 6 3 3 1 1 5 2
52 6 58 4 1 3 1 1
23 14 37 4 2 4 3 5
100 14 114 4 3 3 7 1
72 10 92 4 1 3 3 3
94 12 106 6 2 3 3 1 1 2
31 2 33 2 1 1
27 7 34 6 1 2 1 1 1 1
33 4 37 4 1 1 1 1
1 14 15 5 4 2 2 4 2
50 4 54 4 1 1 1 1
40 3 43 3 1 1 1
80 2 84 2 1 1
77 2 79 2 1 1
40 14 68 4 5 4 3 2
93 12 105 3 5 3 4
36 15 51 7 2 3 2 2 1 3 2
11 6 17 6 1 1 1 1 1 1
9 5 14 3 1 1 3
3 8 11 2 4 4
73 17 107 2 10 7
28 7 42 5 1 1 1 1 3
40 11 51 7 2 1 1 3 1 1 2
101 16 117 7 1 1 5 2 3 1 3
19 4 23 4 1 1 1 1
25 13 51 7 1 2 3 2 1 2 2
85 11 107 7 1 2 2 1 2 2 1
11 7 25 5 1 1 2 1 2
46 9 55 4 2 1 5 1
55 14 83 5 4 3 4 1 2
3 16 19 6 2 3 3 3 3 2
4 18 2 16 9
0
0
0
0
1 37
3 17 4 19
0
2 8 17
3 23 25 17
1 34
0
1 18
3 32 21 40
2 12 18
0
0
0
0
1 4
1 29
2 28 3
2 35 28
2 21 5
2 39 18
0
0
0
0
2 31 26
2 19 29
0
1 8
0
1 17
1 39
0
2 2 16
1 7
0
