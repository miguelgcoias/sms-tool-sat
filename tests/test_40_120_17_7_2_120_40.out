10
2 37 38 39 40 41 42
6 56 57 59 61
8 25 26 27 29
16 31 32
18 33 34 35 36
28 9 10 11
33 101 102 103 108 110 113 114
34 19 20 21 22
37 14 15 16 18 23
38 46 48 49 54
